src/gcp_scanner/crawler/crawler_factory.py

```diff
@@ -55,6 +56,7 @@
   "static_ips": ComputeStaticIPsCrawler,
   "subnets": ComputeSubnetsCrawler,
   "sourcerepos": CloudSourceRepoCrawler,
+  "kms": KMSKeysCrawler,
```

Comment:
minor nit: we try to maintain this list in alphabetical order

Reply:
done :+1: